layouts/partials/tw/footer.html

```diff
@@ -3,122 +3,13 @@
 
 
 <div class="bg-beige font-sans" data-sf-campaign="701Dn000000YgovIAC">
-  <div class="bg-black p-4 sm:py-8 md:py-12">
-    <h2 class="flex items-end justify-center gap-8">
-      <a
-        class="flex w-3/5 max-w-xs text-white hover:text-yellow md:w-1/2 md:max-w-md"
-        href="/"
-      >
-        <svg class="aspect-[16/3] w-full fill-current">
-          <title>Home</title>
-          <desc>Spotlight PA logo</desc>
-          {{ partial "helper/svg.html" "banner" }}
-        </svg>
-      </a>
-    </h2>
-    <h3
-      class="mx-auto my-7 text-center text-lg leading-relaxed tracking-wide text-white sm:max-w-2xl md:my-8 md:px-6 md:text-2xl md:font-bold"
-    >
-      Get the top stories and best investigative journalism from across Pa. with
-      our free newsletters.
-    </h3>
+  {{ partial "tw/newsletter-box-lg.html" (dict
+    "showLogo" "show"
```

Comment:
Since it's just tested with `with` on the other side, this should probably be `true`.